Source/Core/Core/Core.cpp

```diff
@@ -399,7 +399,7 @@ void EmuThread()
 
 	if (!g_video_backend->Initialize(s_window_handle))
 	{
-		PanicAlert("Failed to initialize video backend!");
+		PanicAlertT("Failed to initialize video backend!");
```

Comment:
How is this remotely useful to translate?  A user can't treat this as anything other than "Dolphin isn't working".